swaglyrics/cli.py

```diff
@@ -104,7 +107,7 @@ def lyrics(song: str, artist: str, make_issue: bool = True) -> str:
 	if not lyrics:
 		lyrics = f"Couldn't get lyrics for {song} by {artist}.\n"
 		# Log song and artist for which lyrics couldn't be obtained
-		with open(unsupported_txt, 'a') as f:
+		with open(unsupported_txt, mode='a', encoding='utf-8') as f:
```

Comment:
```suggestion
		with open(unsupported_txt, 'a', encoding='utf-8') as f:
```
same thing but consistency